Backpack/src/main/java/net/skyscanner/backpack/text/BpkTextField.kt

```diff
@@ -79,12 +79,13 @@ open class BpkTextField @JvmOverloads constructor(
   private fun initialize(attrs: AttributeSet?, defStyleAttr: Int) {
     BpkText.getFont(context, BpkText.BASE, BpkText.Weight.NORMAL).applyTo(paint)
 
-    var textColor = ContextCompat.getColor(context, R.color.__textFieldText)
+    var textColor = ContextCompat.getColor(context, R.color.bpkTextPrimary)
+    var textColorDisabled = ContextCompat.getColor(context, R.color.__textFieldTextDisabled)
     var hintNormalColor = ContextCompat.getColor(context, R.color.__textFieldHint)
-    var hintFocusedColor = ContextCompat.getColor(context, R.color.bpkTextSecondary)
+    var hintFocusedColor = ContextCompat.getColor(context, R.color.__textFieldHint)
```

Comment:
I don't see any changes in the focused placeholder color in the new design for now. Therefore lets keep the current API for a while before we remove the attribute completely. By default focused placeholder color matches unfocused placeholder color.